tox.ini

```diff
@@ -1,22 +1,25 @@
 # https://docs.djangoproject.com/en/dev/faq/install/#what-python-version-can-i-use-with-django
 [tox]
 envlist =
-   py{27,34,35}-django110,
-   py{27,34,35,36}-django111,
+    py{27,34,35,36,37}-django111,
+    py{34,35,36,37}-django20,
+    py{35,36,37}-django21,
+skip_missing_interpreters=True
```

Comment:
@Eorhim why skip missing ones? if it's related to the python missing from the build server as not in the distro, that should be solved by the use of the deadsnakes repository (IIRC that was part of the ansible playbook to add/remove that repo during install, just need to add the new python version there too IIRC).